src/encoded/static/components/search.js

```diff
@@ -407,6 +421,12 @@ const ExperimentComponent = (props, reactContext) => {
                                         )}
                                     </div>
                                 )}
+                                {screenLink ?
+                                    <div><strong>candidate Cis-Regulatory Elements (cCREs): </strong><a href={`https://screen.encodeproject.org/search?q=${screenSearch}&assembly=${screenAssembly}`}>SCREEN</a></div>
```

Comment:
<strong> or have a css class to bolden the text?

Reply:
Yes, this would be a lot better. I'll change that on all of these.